spec.bs

```diff
@@ -3975,6 +4066,17 @@ To <dfn>decode an additional bid json</dfn> given a [=string=] |additionalBidJso
         [=decoded additional bid/negative target interest group names=].
   1. Set |result|'s [=decoded additional bid/bid=] to a new [=generated bid=] with the following
     properties:
+    : [=generated bid/reporting id=]
+    :: A [=reporting bid key=] with the following [=struct/items=]:
+      : [=reporting bid key/context=]
+      :: |reportingContextMap|[|auctionConfig|]
+      : [=reporting bid key/source=]
+      :: [=reporting bid source/additional-bid=]
+      : [=reporting bid key/origin=]
+      :: |ig|'s [=interest group/owner=]
+      : [=reporting bid key/interest group name=]
+      :: A string representation of a new globably unique identifier. This is needed since |igName|

```

Comment:
It's interest group name for regular interest groups. For additional bids it can't be, since there is nothing saying that they have to provide a unique interest group name. (And now I am scared of something in our impl breaking because of that...).

Reply:
does impl rely on IG name? if so, we need to fix it?

Reply:
Impl generally uses BidState* as equivalent of "reporting bid key" for reporting. There are a bunch of other things that use InterestGroupKey which seem fine for what I looked, but there are too many for me to be certain none get confused with additionalBid stuff (especially since it's somehow wired into fenced frame?)